instrumentation/buildtime/mapping-file/plugin/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+import utils.artifactPrefix
+import utils.defaultGroupId
+
+private val pluginName = "mapping-file-plugin"
+
+plugins {
+    id("java-gradle-plugin")
+    kotlin("jvm")
+    id("maven-publish")
+    id("signing")
+}
+
+group = defaultGroupId
+version = Configurations.sdkVersionName
+
+val javadocJar by tasks.creating(Jar::class) {
+    archiveClassifier.set("javadoc")
+    from(tasks.named("javadoc"))
+    metaInf {
+        from("$projectDir/src/main/assets/LICENSE.txt")
+    }
```

Comment:
This license file inclusion is probably leftover logic from AppD, and I believe it should not be here for the Splunk Agents. Also, I think there is no `LICENSE.txt` in `"$projectDir/src/main/assets/LICENSE.txt"`. If my assumptions are true, please remove it.